rllib/policy/dynamic_tf_policy.py

```diff
@@ -509,6 +494,8 @@ def fake_array(tensor):
         batch_for_postproc = UsageTrackingDict(sb)
         batch_for_postproc.count = sb.count
         logger.info("Testing `postprocess_trajectory` w/ dummy batch.")
+        self.exploration.postprocess_trajectory(
```

Comment:
Have to also call exploration's postprocessing (may access fields in the batch we need to track; e.g. curiosity).